src/create-dashboard.js

```diff
@@ -19,19 +20,26 @@ module.exports = function (selector, vizJSON, opts) {
   var dashboardEl = document.querySelector(selector);
   if (!dashboardEl) throw new Error('no element found with selector ' + selector);
 
+  // Default options
+  opts = opts || {};
+  opts.renderMenu = _.isBoolean(opts.renderMenu)
+    ? opts.renderMenu
+    : true;
+
   var widgets = new WidgetsCollection();
 
-  var dashboardInfoModel = new cdb.core.Model({
+  var model = new cdb.core.Model({
     title: vizJSON.title,
     description: vizJSON.description,
     updatedAt: vizJSON.updated_at,
     userName: vizJSON.user.fullname,
-    userAvatarURL: vizJSON.user.avatar_url
+    userAvatarURL: vizJSON.user.avatar_url,
+    renderMenu: opts.renderMenu
```

Comment:
IMHO no, because when set to false it do not _render_ (or create the view), so _visible_ would be misleading.